proposals/XXXX-unicode-normalization.md

```diff
@@ -146,6 +146,10 @@ Note that there is no guarantee that two systems will produce _the same_ stabili
 
 The guarantees provided by stabilised strings are likely to be attractive to many Swift developers. In addition to normalisation, it makes sense for the standard library to also offer APIs for producing _stable_ normalisations.
 
+#### Stability over ancient versions of Unicode
+
+Early versions of Unicode experienced significant churn and the modern definition of normalization stability was added in version 4.1. When we talk about stability of assigned code points, we are referring to notions of stability from 4.1 onwards. A normalized string is stable from either the version of Unicode in which the latest code point was assigned or Unicode version 4.1 (whichever one is later).
+
```

Comment:
I think we're going to want more of a discussion about what we mean by "all versions of Unicode" in doc comments.